examples/ocr-serving/copied_file/request_ocr/ocrLibrary/OcrHandle.py

```diff
@@ -0,0 +1,111 @@
+import requests
+from PIL import Image
+from tesserocr import PyTessBaseAPI, RIL
+import pyocr
+import pyocr.builders
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def getOcrMethod(methodName):
+    if methodName == "tesserocr":
+        return useOcrTesserocr
+    elif methodName == "pyocr":
+        return useOcrPyocr
+    elif methodName == "api":
+        return useOcrApi
+    else:
+        logger.error("invalid methodName: {}".format(methodName))
+        return None
+
+
+def useOcrTesserocr(imgPath):
```

Comment:
tesserocr & pyocr are all python wrapper of Tesseract ocr model, why use two of them?

Reply:
There is a little performance difference for this two wrapper, I think it is caused by some default values, but no detailed docs for it, so I keep them both.